intro-to-rails-3/controller-filters-contd.md

```diff
@@ -43,5 +45,49 @@ skip_before_action :require_login, except: [:current_user]
 
 The `skip_before_action` ensures that the `require_login` method is **skipped on every action defined in this controller,** *except* the action listed after `except`: the `current_user` action.
 
+<!-- >>>>>>>>>>>>>>>>>>>>>> BEGIN CHALLENGE >>>>>>>>>>>>>>>>>>>>>> -->
+<!-- Replace everything in square brackets [] and remove brackets  -->
+
+### !challenge
+
+* type: short-answer
+* id: 3dbe2363-c73d-4cd5-81fc-83c410cb2cdd
+* title: skip_before_action location
+<!-- * points: [1] (optional, the number of points for scoring as a checkpoint) -->
+<!-- * topics: [python, pandas] (optional the topics for analyzing points) -->
+
+##### !question
+
+Why is the `skip_before_action` located in the `UsersController` when `before_action` is located in the `ApplicationController`?
+
+##### !end-question
+
+##### !placeholder
+
+Your answer here
+
+##### !end-placeholder
+
+##### !answer
+
+/.+/
+
+##### !end-answer
+
+<!-- other optional sections -->
+<!-- !hint - !end-hint (markdown, users can see after a failed attempt) -->
+<!-- !rubric - !end-rubric (markdown, instructors can see while scoring a checkpoint) -->
+##### !explanation
+
+We want the before filter to run at the beginning of every action in the application (in every controller), **except** for the `current_user` action located specifically in the UsersController. 
+In other words, we do this because we want to filter to apply universally to the application and we want the exception to apply in a very specific instance.
```

Comment:
```suggestion
In other words, we do this because we want to filter to apply universally to the application and we want the exception to apply in a very specific instance.  This works because **every** controller in our application inherits from `ApplicationController`.
```